Assets/Scripts/Minigames/ShipRepairs/GreenZone.cs

```diff
@@ -14,7 +14,8 @@ private void Start()
     // Call this to increase the difficulty 
     public void ReduceSize()
     {
-        transform.localScale += new Vector3(-RepairsConstants.sizeDecrease, 0f, 0f); 
+        transform.localScale += new Vector3(
```

Comment:
why not just use -=

Reply:
honestly i normally do it this way 
no idea why and cant really be bothered it to change it